### PR TITLE
Pia 3855 bank extractions edit

### PR DESCRIPTION
**Pull request by llevente**

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Components/DigitalLineItemTableViewCell.swift [hunk 1]
```diff
@@ -48,7 +48,7 @@ class DigitalLineItemTableViewCell: UITableViewCell {
             modeSwitch.onTintColor = viewModel.modeSwitchTintColor
 
             [nameLabel, editButton, unitPriceLabel, priceLabel].forEach { view in
-                view.alpha = viewModel.lineItem.selectedState == .selected ? 1 : 0.5
+                view?.alpha = viewModel.lineItem.selectedState == .selected ? 1 : 0.5
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Components/DigitalLineItemTableViewCell.swift:51]: [Clarification] @llevente why do we need this change?

**Review by zladzeyka — COMMENTED** [review]
Looks good! 👍 
I have only a tiny question about the change in the SDK source code.

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Components/DigitalLineItemTableViewCell.swift (hunk 1 above)
Comment (llevente) [reply BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Components/DigitalLineItemTableViewCell.swift:51]: My XCode gives me build error that the optional "view" must be unwrapped after merging new-ui into my branch. Haven't worked on these classes, it was strange also for me. When i started working on the task it ran without error

**Review by zladzeyka — APPROVED** [review]
Thank you @llevente

**Pull request merged**